src/instructlab/generator/README.md

```diff
@@ -1,7 +1,8 @@
 ### Notice
-This folder uses code from the `stanford_alpaca` project developed by Tatsu Lab, available at https://github.com/tatsu-lab/stanford_alpaca. The original code is licensed under the Apache License:
 
-```
+This folder uses code from the `stanford_alpaca` project developed by Tatsu Lab, available [here](https://github.com/tatsu-lab/stanford_alpaca). The original code is licensed under the Apache License:
+
+```license
```

Comment:
Do you know for sure this is valid?  I don't think it'll hurt if it's not recognized. I would have just done `text` probably. I won't block on it since I don't think it breaks anything

Reply:
I see it being used other places in the repo, so I assume it's valid